test/Microsoft.Health.Dicom.Tests.Integration/Persistence/ChangeFeedTests.cs

```diff
@@ -129,6 +153,10 @@ private async Task ValidateInsertFeedAsync(VersionedInstanceIdentifier dicomInst
         Assert.Equal(expectedCount, result.Count);
         Assert.Equal((int)ChangeFeedAction.Create, result.Last().Action);
         Assert.Equal(result.Last().OriginalWatermark, result.Last().CurrentWatermark);
+        if (expectedFileProperties is not null)
```

Comment:
Couldn't we say: `Assert.Equal(expectedFileProperties, result.Last().FilePath)`?